src/magicbox/SuggestionsManager.ts

```diff
@@ -350,17 +316,18 @@ export class SuggestionsManager {
   }
 
   private async moveWithinSuggestion(direction: Direction) {
-    const currentlySelected = $$(this.element).find(`.${this.options.selectedClass}`);
-    const selectables = $$(this.element).findAll(`.${this.options.suggestionClass}`);
-    const currentIndex = indexOf(selectables, currentlySelected);
+    const currentlySelected =
+      this.keyboardFocusedElement && this.keyboardFocusedElement.type === 'suggestion' ? this.keyboardFocusedElement.suggestion : null;
+    const selectables = this.displayedSuggestions;
+    const currentIndex = currentlySelected !== null ? indexOf(selectables, currentlySelected) : -1;
```

Comment:
`currentlySelected !== null` -> `currentlySelected`

Reply:
Good point, fixing that one too.